specification/loadtestservice/LoadTestService.Management/models.tsp

```diff
@@ -0,0 +1,233 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-providerhub";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+using OpenAPI;
+using Autorest;
+using Azure.ResourceManager.Foundations;
+
+namespace Microsoft.LoadTestService.Draft;
+
+@doc("LoadTest details.")
+model LoadTestResource is TrackedResource<LoadTestProperties> {
+  @doc("Load Test name")
+  @key("loadTestName")
+  @path
+  @segment("loadTests")
+  name: string;
+}
+
+@doc("Resources provisioning states.")
+enum ResourceState{
```

Comment:
This is not defined already? Is it something we are defining differently?

Reply:
We have a ResourceProvisioningState in library but it does not have Deleted state

```
enum ResourceProvisioningState {
  @doc("Resource has been created.")
  Succeeded,

  @doc("Resource creation failed.")
  Failed,

  @doc("Resource creation was canceled.")
  Canceled,
}
```